src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_authn_ldap.erl

```diff
@@ -197,20 +201,27 @@ result_to_user_ejson(LoginAttr, UserName, [{eldap_entry, CN, DataIn}|_]) ->
     %
     % Note that any missing fields in the user's directory entry (LookupFields)
     % will have "unknown" substituted in the returned json record.
-    LookupFields = [{"displayname", <<"display_name">>},
-                    {"givenname", <<"first_name">>},
-                    {"sn", <<"last_name">>},
-                    {"cn", <<"common_name">>},
-                    {"c", <<"country">>},
-                    {"l", <<"city">>},
-                    {"mail", <<"email">>} ],
-
-    Terms = [ {Name, value_of(Key, Data, "unknown") } || {Key, Name} <- LookupFields ],
+    Terms = [ {Name, value_of(Key, Data, "unknown") } || {Key, Name} <- ldap_attribute_map() ],
     Result = Terms ++ [ { <<"username">>, CanonicalUserName },
                         { <<"external_authentication_uid">>, UserName },
                         { <<"recovery_authentication_enabled">>, false } ],
     {CN, CanonicalUserName, {Result}}.
 
+
+ldap_attribute_map() ->
+    Config = envy:get(oc_chef_wm, ldap, list),
+    Map = [{proplists:get_value(display_name_attribute, Config, "displayname"), <<"display_name">>},
```

Comment:
It's a little curious to have the defaults be there twice... or am I missing something? We shouldn't end up with a config that doesn't have at least the default values in it... (But it doesn't hurt, I guess)